modin/polars/dataframe.py

```diff
@@ -0,0 +1,1340 @@
+
+    def get_column_index(self, name: str) -> int:
+        """
+        Find the index of the column by name.
+
+        Args:
+            name: Name of the column to find.
+
+        Returns:
+            Index of the column.
+        """
+        return self.columns.index(name)
+
+    find_column_index_by_name = get_column_index
+
+    def get_columns(self) -> list["Series"]:
+        """
+        Get the columns of the DataFrame.
+
+        Returns:
+            List of Series with the columns.
+        """
+        return [self[name] for name in self.columns]
+
+    def group_by_dynamic(
+        self,
+        index_column,
+        *,
+        every,
+        period,
+        offset,
+        truncate,
+        include_boundaries,
+        closed,
+        label,
+        group_by,
+        start_by,
+        check_sorted,
+    ):
+        raise NotImplementedError("not yet")
+
+    def group_by_rolling(
+        self,
+        index_column,
+        *,
+        every,
+        period,
+        closed,
+        offset,
+        by,
+        check_sorted,
+    ):
+        raise NotImplementedError("not yet")
+
+    def hstack(self, columns, *, inplace: bool = False) -> "DataFrame":
+        """
+        Stack the given columns horizontally.
+
+        Args:
+            columns: Columns to stack.
+            inplace: Whether to stack the columns in place.
+
+        Returns:
+            DataFrame with the columns stacked horizontally.
+        """
+        if isinstance(columns, DataFrame):
+            columns = columns.get_columns()
+        result_query_compiler = self._query_compiler.concat(
+            axis=1, other=[c._query_compiler for c in columns]
+        )
+        if inplace:
+            self._query_compiler = result_query_compiler
+            return self
+        return self.__constructor__(_query_compiler=result_query_compiler)
+
+    def insert_column(self, index: int, column: "Series") -> "DataFrame":
+        """
+        Insert the given column at the given index.
+
+        Args:
+            index: Index to insert the column at.
+            column: Column to insert.
+            name: Name of the column to insert.
+
+        Returns:
+            DataFrame with the column inserted.
+        """
+        return self.__constructor__(
+            self._query_compiler.insert(index, column.name, column._query_compiler)
+        )
+
+    def item(self, row: int | None = None, column: str | int | None = None) -> Any:
+        """
+        Get the value at the given row and column.
+
+        Args:
+            row: Row to get the value from.
+            column: Column to get the value from.
+
+        Returns:
+            Value at the given row and column.
+        """
+        if row is None:
+            row = 0
+        if column is None:
+            column = 0
+        if isinstance(column, str):
+            column = self.columns.index(column)
+        return self.to_pandas().iloc[row, column]
```

Comment:
can we select the item before converting to pandas?

Reply:
`to_pandas` in this case converts to `modin.pandas.DataFrame`